src/firebase.js

```diff
@@ -11,8 +11,25 @@ var firebaseConfig = {
   messagingSenderId: "975187077657",
   appId: "1:975187077657:web:9bef6448123610a32d5888",
 };
+
+var firebaseTestConfig = {
+  apiKey: "AIzaSyDXWnA0fG68qydK6204CCrf6CXOpgwI-j8",
```

Comment:
We should work on extracting this in a separate step

Reply:
we need it for now, Atem and Christian are fixing it and then we fix the test